op-program/client/tasks/deposits_block.go

```diff
@@ -0,0 +1,138 @@
+package tasks
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-program/client/l1"
+	"github.com/ethereum-optimism/optimism/op-program/client/l2"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/consensus/misc/eip1559"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+// BuildDepositOnlyBlock builds a deposits-only block replacement for the specified optimistic block and returns the block hash and output root
+// for the new block.
+// The specified l2OutputRoot must be the output root of the optimistic block's parent.
+func BuildDepositOnlyBlock(
+	logger log.Logger,
+	cfg *rollup.Config,
+	l2Cfg *params.ChainConfig,
+	optimisticBlock *types.Block,
+	optimisticBlockOutput *eth.OutputV0,
+	l1Head common.Hash,
+	agreedL2OutputRoot eth.Bytes32,
+	l1Oracle l1.Oracle,
+	l2Oracle l2.Oracle,
+) (common.Hash, eth.Bytes32, error) {
+	engineBackend, err := l2.NewOracleBackedL2Chain(logger, l2Oracle, l1Oracle /* kzg oracle */, l2Cfg, common.Hash(agreedL2OutputRoot))
```

Comment:
Why the `kzg oracle` comment here?

Reply:
leftover copy from a similar comment in https://github.com/ethereum-optimism/optimism/blob/develop/op-program/client/tasks/derive.go#L42. Will remove.